extensions/sql-database-projects/src/controllers/databaseProjectTreeViewProvider.ts

```diff
@@ -71,7 +78,13 @@ export class SqlDatabaseProjectTreeViewProvider implements vscode.TreeDataProvid
 		const projNode = this.roots.find(x => x instanceof ProjectRootTreeItem ? (<ProjectRootTreeItem>x).project === project : false);
 
 		if (projNode) {
-			this.treeView?.reveal(projNode, { focus: true, expand: true });
+			await this.treeView?.reveal(projNode, { focus: true, expand: true });
+		}
+	}
+
+	public async expandAllNodes(): Promise<void> {
```

Comment:
it's a little confusing that this is called "Expand All" but it's only expanding the first level and not all the folders. What's the Visual Studio behavior for this?

Reply:
ah makes sense... Visual Studio does not actually have an expand all option for file explorer, only collapse all. Same with Notebooks and Explorer in ADS

Reply:
hmm... I thought VS has both... am I perceiving this wrong
![image](https://user-images.githubusercontent.com/46980425/89928737-225c5600-dbbd-11ea-8cf6-e570952b6b92.png)

Reply:
the "Show All Files" button shows the generated files and stuff not part of the project, not expanding the folders